tic_tac_toe/game.py

```diff
@@ -63,22 +83,50 @@ def _check_winning_combinations(board, player):
     Returns the player (winner) of any of the winning combinations is completed
     by given player, or None otherwise.
     """
-    pass
+    first_row = ((0, 0), (0, 1), (0, 2))
+    second_row = ((1, 0), (1, 1), (1, 2))
+    third_row = ((2, 0), (2, 1), (2, 2))
+    first_col = ((0, 0), (1, 0), (2, 0))
+    second_col = ((0, 1), (1, 1), (2, 1))
+    third_col = ((0, 2), (1, 2), (2, 2))
+    diag_1 = ((0, 0), (1, 1), (2, 2))
+    diag_2 = ((0, 2), (1, 1), (2, 0))
+    winning_combinations  = [first_row, second_row, third_row, first_col, second_col, third_col, diag_1, diag_2]
```

Comment:
@jsymons That is how I had written it before, but found that it was easier for me to visualize/write the logic with the tuples named this way.